examples/chart/teleport-kube-agent/values.yaml

```diff
@@ -242,6 +242,19 @@ log:
 # https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
 
+# Pod's DNS Configuration
+# https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config
+# This value is useful if you need to reduce the DNS load: set "ndots" to 0 and only use FQDNs.
+dnsConfig: {}
```

Comment:
Since we included `dnsConfig`, might be good to include the `dnsPolicy` as well.

Reply:
done